FindYourMovie/Controllers/MovieViewController.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  ViewController.swift
+//  FindYourMovie
+//
+//  Created by Andres Guevara Caprio on 8/18/20.
+//  Copyright © 2020 Andres Guevara Caprio. All rights reserved.
+//
+
+import UIKit
+
+/// Controller for main view. It is a Table View that contains a search controller to search for movies through a movie databse API.
+/// The last 10 movies are persisted locally and displayed as a suggestion when the user searches for movies
+class MovieViewController: UITableViewController {
+    
+    var movieSearch = [Movie]()
+    var currentPage = 1
+    var currentSearch = ""
+    var totalPages = 1
+    var last10searches = [String]()
+    var lastSearchesController : MovieSearchController!
+    var searchController : UISearchController!
+    let defaults = UserDefaults.standard
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Create an instance of the second controller once main controller loads
+        lastSearchesController = MovieSearchController()
+        // Make the second controller a search result controller
+        searchController = UISearchController(searchResultsController: lastSearchesController)
+        searchController.searchResultsUpdater = self
+        if #available(iOS 13.0, *) {
+            searchController.searchBar.searchTextField.placeholder = "Search for a Movie"
+        } else {
+            searchController.searchBar.placeholder = "Search for a Movie"
+        }
+        // Add search controller to navigation controller
+        navigationItem.searchController = searchController
+        navigationItem.hidesSearchBarWhenScrolling = false
+        searchController.delegate = self
+        searchController.searchBar.delegate = self
+        definesPresentationContext = true
+        // Load latestes searches at loading time
+        if let searches = self.defaults.array(forKey: "lastSearches") as? [String] {
+            self.last10searches = searches
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return movieSearch.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "MovieCell", for: indexPath) as! MovieCell
+        // Configure each cell
+        let movie = movieSearch[indexPath.row]
```

Comment:
To keep the View Controller cleaner, we could have a `configure()` func on the MovieCell, which when passed a movie, will assign these items. This would keep our cellForRow much shorter, and allow the Cell to handle its own setting of UI elements. Taking it a step further, the cell could have a view model as well. VM not _required_ for this project though.